src/meta/src/hummock/manager/compaction_group_manager.rs

```diff
@@ -605,16 +605,6 @@ impl HummockManager {
         drop(compaction_guard);
         self.report_compact_tasks(canceled_tasks).await?;
 
-        // Don't trigger compactions if we enable deterministic compaction
```

Comment:
after https://github.com/risingwavelabs/risingwave/pull/18053/commits/2ccdb93809f65596b1aade9d42b48a9f900d0a16

SstableInfo's `table_ids` have already been modified in `split_sst`, so there is no need to trigger SpaceRecliam.

Reply:
What if all the table ids of a SST is split out? I think in our current implementation, we will have two SST: one with empty table id and the other one with identical table id as before the split.

Reply:
SstableInfo with empty table_ids will be cleaned up immediately in the apply_version_delta phase.